src/assets/style/app.scss

```diff
@@ -1,3 +1,9 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import "fonts.scss";
+
+
+body {
+    @apply overflow-x-hidden font-inter text-[16px];
```

Comment:
I agree with @markoMVP85 about overflow. About text-[16px], we using it because we want to use tailwind spacing, and this definition is just to have scaling 1rem = 16px.